source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +354,122 @@ the steps described in :ref:`debugging-deployer-issues`
   Changing or deleting a processed commit file could cause unchanged files to be indexed again and
   it should be done as a last resort in case of errors.
 
+^^^^^^^^^^^^^^^^^
+Document Indexing
+^^^^^^^^^^^^^^^^^
+Crafter Deployer indexes documents via the following configured items:
```

Comment:
While the Deployer performs content indexing, we probably need to reference this higher up in the IA and make it clear(er). Let's discuss where else to point to this or include it. This is core to the system.

Reply:
This is where to discuss this: https://docs.craftercms.org/en/4.1/by-role/developer/common/search.html#content-search